discord/src/main/resources/lang/English (UK).yml

```diff
@@ -19,7 +19,7 @@ values:
   couldnt-find-setting: ":x: Could not find that setting. Use `>preset settings` if you're unsure."
   no-active-giveaways: ":x: There are no active giveaways in this server."
   no-scheduled-giveaways: ":x: There are no scheduled giveaways in this server."
-  fatal-error-loading-server: ":warning: There was a fatal error loading your server. Please join the support discord and scream at Zak."
+  fatal-error-loading-server: ":warning: There was a fatal error loading your server. Please join the support discord at https://discord.gg/ancjpjEEPX and scream at Zak."
```

Comment:
pls do <> and put the link inside it

like https://moron.com

Reply:
Fixed in 80a7cfc387e689e32f0a1c1f8100a96378ca0d03